settings.gradle.kts

```
@@ -91,6 +91,8 @@ dependencyResolutionManagement {
 
 include("commons")
 
+include("s3-connectors-common")
```

```suggestion
include("s3-common")
```
As it's a utils module and not connector